migrations/versions/426f1ad50ca2_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 426f1ad50ca2
+Revises: 598562686e8
+Create Date: 2015-02-18 15:46:44.779144
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '426f1ad50ca2'
+down_revision = '598562686e8'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('items_users',
```

Comment:
Followed the previous convention by which the role-user relationship is called role_user